src/mlpack/tests/activation_functions_test.cpp

```diff
@@ -406,6 +407,55 @@ BOOST_AUTO_TEST_CASE(SELUFunctionDerivativeTest)
       selu.Lambda() * selu.Alpha() - arma::mean(activations))), 10e-4);
 }
 
+/*
+ * Implementation of the CELU activation function test. The function is
+ * implemented as CELU layer in the file celu.hpp
+ *
+ * @param input Input data used for evaluating the CELU activation function.
+ * @param target Target data used to evaluate the CELU activation.
+ */
+void CheckCELUActivationCorrect(const arma::colvec input,
```

Comment:
Hey, @gaurav-singh1998 There is no need to add new function to test celu you can use `CheckActivationCorrect` and `CheckDervativeCorrect`.

Reply:
Agreed, You can refer to test of any other activation function.